### PR TITLE
Force 3 station replications

### PR DESCRIPTION
**Pull request by shohamroditimemphis**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -123,6 +123,11 @@ func runMemphis(s *server.Server) {
 		s.Errorf("Failed initializing integrations: " + err.Error())
 	}
 
+	err = s.Force3ReplicationsPerStation()
```
Comment (idanasulinmemphis) [inline main.go:126]: Force3ReplicationsPerStation => Force3ReplicationsForExistingStations

File: main.go [hunk 2]
```diff
@@ -123,6 +123,11 @@ func runMemphis(s *server.Server) {
...
+	if err != nil {
+		s.Errorf("Failed forece 3 replications per station: " + err.Error())
```
Comment (idanasulinmemphis) [inline main.go:128]: forece => force .. replications for existing stations

File: server/memphis_handlers_user_mgmt.go [hunk 3]
```diff
@@ -373,8 +373,8 @@ func (umh UserMgmtHandler) AddUserSignUp(c *gin.Context) {
 
 	newUser, err := db.CreateUser(username, "management", hashedPwdString, fullName, subscription, 1, MEMPHIS_GLOBAL_ACCOUNT, false, "", "", "", "")
 	if err != nil {
-		if strings.Contains(err.Error(), "already exist") {
-			c.AbortWithStatusJSON(SHOWABLE_ERROR_STATUS_CODE, gin.H{"message": "User already exist"})
+		if strings.Contains(err.Error(), "already exists") {
```
Comment (idanasulinmemphis) [inline server/memphis_handlers_user_mgmt.go:376]: return to exist

File: ui_src/src/components/createStationForm/index.js [hunk 4]
```diff
@@ -286,31 +287,32 @@ const CreateStationForm = ({ createStationFormRef, getStartedStateRef, finishUpd
...
-                                popupClassName="select-options"
-                                options={actualPods}
-                                value={getStartedStateRef?.formFieldsCreateStation?.replicas || actualPods[0]}
-                                onChange={(e) => getStarted && updateFormState('replicas', e)}
-                                disabled={!allowEdit}
-                            />
-                        </Form.Item>
-                    </div>
-                </div>
+                {isCloud() ? null :
```
Comment (avrhamNeeman) [inline ui_src/src/components/createStationForm/index.js:290]: Replace it with !isCloud() && <div ....

File: ui_src/src/services/valueConvertor.js [hunk 5]
```diff
@@ -337,6 +337,9 @@ export const tieredStorageTimeValidator = (value) => {
 };
 
 export const replicasConvertor = (value, stringToNumber) => {
+    if (isCloud()) {
```
Comment (avrhamNeeman) [inline ui_src/src/services/valueConvertor.js:340]: Why you didn't pass it by param?

**Review by idanasulinmemphis — APPROVED** [review]

File: ui_src/src/components/createStationForm/index.js (hunk 4 above)
Comment (shohamroditimemphis) [reply ui_src/src/components/createStationForm/index.js:290]: resolve in this pr: https://github.com/memphisdev/memphis/pull/1091

File: ui_src/src/services/valueConvertor.js (hunk 5 above)
Comment (shohamroditimemphis) [reply ui_src/src/services/valueConvertor.js:340]: resolve in this pr:https://github.com/memphisdev/memphis/pull/1091

**Pull request merged**